test/blackbox-tests/test-cases/pkg/pkg-action-when.t

```diff
@@ -0,0 +1,29 @@
+Testing the when action in lockfiles
+
+  $ mkdir dune.lock
+  $ cat >dune.lock/lock.dune <<EOF
+  > (lang package 0.1)
+  > EOF
+
+Case with a mix of uncoditional and conditional actions in a progn action
+  $ cat >dune.lock/test.pkg <<'EOF'
+  > (install
+  >  (progn
+  >   (run echo foo)
+  >   (when (= foo foo)
+  >    (run echo bar))
+  >   (when (= foo bar)
+  >    (run echo baz))))
```

Comment:
Can you add a test for an operation that isn't `=`? Just for some variety, to see it work properly evaluated.

Reply:
done